ext/lmdb-safe/lmdb-safe.hh

```diff
@@ -1,4 +1,7 @@
 #pragma once
+#include <bits/stdint-uintn.h>
```

Comment:
on macOS:

```
../../ext/lmdb-safe/lmdb-safe.hh:2:10: fatal error: 'bits/stdint-uintn.h' file not found
```

according to https://stackoverflow.com/a/65983825/297793:

`#include <stdint.h>` is the correct way to declare these types. Any other approach is undefined and/or non-portable.

Reply:
silly, clang tooling invented this #include for me :(

adjusted.